src/primitives/point.jl

```diff
@@ -49,7 +49,7 @@ paramdim(::Type{<:Point}) = 0
 
 center(p::Point) = p
 
-==(A::Point, B::Point) = A.coords == B.coords
+==(A::Point, B::Point) = to(A) == to(B)
```

Comment:
can you please confirm that this is as fast as before with Cartesian coords?

Reply:
Yes, it is:
```
julia> using Meshes

julia> p = Point(1, 2)
Point with Cartesian{NoDatum} coordinates
├─ x: 1.0 m
└─ y: 2.0 m

julia> test1(A::Point, B::Point) = A.coords == B.coords
test1 (generic function with 1 method)

julia> test2(A::Point, B::Point) = to(A) == to(B)
test2 (generic function with 1 method)

julia> using BenchmarkTools

julia> @btime test1($p, $p)
  2.114 ns (0 allocations: 0 bytes)
true

julia> @btime test2($p, $p)
  2.121 ns (0 allocations: 0 bytes)
true
```